static/app/components/superuserAccessForm.tsx

```diff
@@ -159,40 +186,65 @@ class SuperuserAccessForm extends Component<Props, State> {
   }
 
   render() {
-    const {authenticators, error, errorType, showAccessForms} = this.state;
+    const {authenticators, error, errorType, showAccessForms, isLoading} = this.state;
     if (errorType === ErrorCodes.INVALID_SSO_SESSION) {
       this.handleLogout();
       return null;
     }
+
     return (
       <ThemeAndStyleProvider>
-        <Form
-          submitLabel={t('Continue')}
-          onSubmit={this.handleSubmit}
-          initialData={{isSuperuserModal: true}}
-          extraButton={
-            <BackWrapper>
-              <Button type="submit" onClick={this.handleSubmitCOPS}>
-                {t('COPS/CSM')}
-              </Button>
-            </BackWrapper>
-          }
-          resetOnError
-        >
-          {error && (
-            <StyledAlert type="error" showIcon>
-              {errorType}
-            </StyledAlert>
-          )}
-          {showAccessForms && <Hook name="component:superuser-access-category" />}
-          {!showAccessForms && (
-            <U2fContainer
-              authenticators={authenticators}
-              displayMode="sudo"
-              onTap={this.handleU2fTap}
-            />
-          )}
-        </Form>
+        {this.props.hasStaff ? (
+          isLoading ? (
+            <LoadingIndicator />
+          ) : (
+            <React.Fragment>
+              {error && (
+                <StyledAlert type="error" showIcon>
+                  {errorType}
+                </StyledAlert>
+              )}
+              {!authenticators.length && !error && (
+                <StyledAlert type="error" showIcon>
+                  {ErrorCodes.NO_AUTHENTICATOR}
+                </StyledAlert>
```

Comment:
We need to show an error immediately if the user has no U2F.

This should only happen if the other error is not set, or else we can end up showing 2 error messages